cln-grpc/proto/node.proto

```diff
@@ -71,6 +71,7 @@ message GetinfoResponse {
 	string lightning_dir = 9;
 	uint32 blockheight = 11;
 	string network = 12;
+	optional uint64 msatoshi_fees_collected = 18;
```

Comment:
Notice that `optional` is no longer supported in proto files, and will cause some compilers to fail.

Reply:
Also was this edited manually? I can't find a place where the `optional` annotation comes from.

Reply:
It is not edited manually, it seems that `optional` has already been used to represent optional fields (see e.g. https://github.com/ElementsProject/lightning/blob/0abe2e3af1ccd9941992266b65f85aa32d3ccb23/cln-grpc/proto/node.proto#L77)

Although this seems not directly related to this PR, I can change `grpc.py` to stop using it if you want me to.
See commit  3e94f7ffb58e7639028dd34ded33ccdc9066fe11 for the change. I will drop if you don't like it.

Reply:
Hm, my bad, let's keep the `optional` for now, otherwise we end up changing more semantics than absolutely required. Sorry :disappointed:

Reply:
Dropped https://github.com/ElementsProject/lightning/commit/3e94f7ffb58e7639028dd34ded33ccdc9066fe11
And rebased.